lib/ruby_indexer/lib/ruby_indexer/index.rb

```diff
@@ -91,10 +91,10 @@ class Entry
       sig { returns(YARP::Location) }
       attr_reader :location
 
-      sig { returns(T::Array[String]) }
+      sig { returns(String) }
```

Comment:
We have an example in `rbi` to find `@` annotations: https://github.com/Shopify/rbi/blob/main/lib/rbi/model.rb#L98-L104.

Since we already get lines, it seems less wasteful to just keep them and call `join` when you need the full block instead of joining during the collect then calling `lines` when you need the lines.

Reply:
That's fair! I reverted back to an array.